src/index.ts

```diff
@@ -71,9 +71,12 @@ async function setDebugSession(
   client: IClientSession | Session.ISession
 ) {
   if (!debug.session) {
-    debug.session = new DebugSession({ client: client });
-  } else {
-    debug.session.client = client;
+    debug.session = new DebugSession();
+  }
+  debug.session.client = client;
```

Comment:
This follows the same approach as when we set the session on the service.

It removes the possibility to provide a `client` to the constructor, so the client can only be set via the setter.